catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/UpdateOperations.java

```diff
@@ -510,6 +510,11 @@ private UpdateResponse processPostIngestPlugins(UpdateResponse updateResponse) {
 
     private UpdateResponse performRemoteUpdate(UpdateRequest updateRequest,
             UpdateResponse updateResponse) {
+        // All metacards were updated locally
+        if (updateResponse != null && updateRequest.getUpdates().size() == updateResponse.getUpdatedMetacards().size()) {
```

Comment:
`updateRequest.getUpdates()` is validated before this method : https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/UpdateOperations.java but it looks like `getUpdatedMetacards` can be null at this point.